assets/queries/terraform/azure/network_interfaces_ip_forwarding_enable/test/positive2.tf

```diff
@@ -0,0 +1,11 @@
+resource "azurerm_network_interface" "positive2" {
+  name                = "example-nic"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  ip_configuration {
+    name                          = "internal"
+    subnet_id                     = azurerm_subnet.example.id
+    private_ip_address_allocation = "Dynamic"
+  }
+}
```

Comment:
I think this file should be a negative, since the default value for `enable_ip_forwarding` is `false`